src/core/storage/fileio/s3_api.cpp

```diff
@@ -346,249 +397,268 @@ list_objects_response list_objects_impl(s3url parsed_url,
     request.WithDelimiter("/"); // seperate objects from directories
 
     bool moreResults = false;
-    do
-    {
+    bool success = false;
+    int backoff = 50;  // ms
```

Comment:
Could you add a comment about your reasoning behind this number here?

Reply:
no reason. Get this number by running the actual program.